### ISSUE TITLE
Zendesk: Group not being assigned when creating a ticket

### ISSUE THREAD

**Issue by marlom-rvvup** [n8n-io/n8n#6491]
**Describe the bug**
We are triggering a "zendesk ticket create" which we want to assign to a specific group, but n8n is not updating it.

**To Reproduce**

1. Import the workflow below.
2. Set up a valid Zendesk API Token on credentials.
3. Select/add a valid group id on Zendesk node.
4. Send it.

<details>
<summary>Workflow example</summary>

```json
{
  "name": "Error example",
  "nodes": [
    {
      "parameters": {
        "httpMethod": "POST",
        "path": "/error-logging",
        "responseMode": "responseNode",
        "options": {}
      },
      "id": "ec9abce3-2a4f-45b2-a5e8-480a82259cd7",
      "name": "Webhook",
      "type": "n8n-nodes-base.webhook",
      "typeVersion": 1,
      "position": [
        780,
        560
      ],
      "webhookId": "81d084f9-db5f-4e7f-88c2-05d7cc92a0b1"
    },
    {
      "parameters": {
        "respondWith": "noData",
        "options": {
          "responseCode": 201
        }
      },
      "id": "fa60f692-5756-4cec-9523-c6c88b45a974",
      "name": "Respond to Webhook1",
      "type": "n8n-nodes-base.respondToWebhook",
      "typeVersion": 1,
      "position": [
        1240,
        560
      ]
    },
    {
      "parameters": {
        "description": "=My ticket description",
        "additionalFields": {
          "group": 13480966960273,
          "subject": "=Something failed"
        }
      },
      "id": "c4627b55-14cc-488c-bcdd-b9d5cf1c2bb0",
      "name": "Zendesk",
      "type": "n8n-nodes-base.zendesk",
      "typeVersion": 1,
      "position": [
        1020,
        560
      ],
      "credentials": {
        "zendeskApi": {
          "id": "27",
          "name": "Zendesk API"
        }
      }
    }
  ],
  "pinData": {},
  "connections": {
    "Webhook": {
      "main": [
        [
          {
            "node": "Zendesk",
            "type": "main",
            "index": 0
          }
        ]
      ]
    },
    "Zendesk": {
      "main": [
        [
          {
            "node": "Respond to Webhook1",
            "type": "main",
            "index": 0
          }
        ]
      ]
    }
  },
  "active": false,
  "settings": {},
  "versionId": "e692e3e9-1035-4469-850e-493d303aebe4",
  "id": "467857693",
  "meta": {
    "instanceId": "4a76d51d29a71c5b99488f8a7cce39b7774a5946d79d1bf7ecaf14e675bdeaed"
  },
  "tags": []
}
```
</details>

**Expected behavior**

We set the group id to `13480966960273` on n8n which is expected to be set on Zendesk, but the group is simply ignored which looks like n8n might be sending the wrong parameter or none at all for the group.

**Environment (please complete the following information):**

- OS: MacOS 13.4
- n8n Version 0.230.3
- Node.js Version 18
- Database system Postgres
- Operation mode own

**Additional context**

Have tried directly to Zendesk API and it does set the group id properly.

Call to `/api/v2/tickets.json` with the body:

```json
{
    "ticket": {
        "subject": "Direct API call test",
        "comment": { "body": "This is a sample comment" },
        "group_id": 15975450236049
    }
}
```

**Comment by Joffcom** [comment]
Hey @marlom-rvvup,

When you are selecting the group id are you picking it from the dropdown list or typing it in? If you are typing it in can you make sure the field is set to an expression and see if that works?

**Comment by Joffcom** [comment]
Hey @marlom-rvvup,

Signed up for a test account and hit the same issue, I have put in a PR that will fix this but I may have missed the cut off for the next release so it may be in `0.234.1` or `0.235.0` depending on timings.

**Comment by Joffcom** [comment]
Hey @marlom-rvvup,

Good news, This change was released in the `0.234.0` release today so an update to this should sort it out. I am going to get this one marked as closed, If you are still seeing an issue after updating let me know and we can open this again.

**Comment by janober** [comment]
Fix got released with [`n8n@0.234.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%400.234.0)
